src/molecules/formfields/RadioField/index.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { camelCase } from 'lodash';
+import classnames from 'classnames';
+
+import styles from './radio_field.module.scss';
+
+function RadioField(props) {
+  const {
+    label,
+    input,
+  } = props;
+
+  return (
+    <label
+      className={classnames(styles['radio-field'])}
+      htmlFor={`radio-${camelCase(input.value)}`}
+    >
+      <input
+        type='radio'
+        name={input.name}
+        checked={input.checked}
```

Comment:
I don't think redux-form will give us `input.selected` so you'll have to do `checked={input.value}`